### DISCUSSION TITLE
Weird error

### DISCUSSION THREAD

**Discussion by yong-shao-eptura** [apache/airflow#21152] [General]
![image](https://user-images.githubusercontent.com/2894545/151360103-7defe27e-6a35-4f26-8a69-e9c0765874e6.png)
![image](https://user-images.githubusercontent.com/2894545/151360202-b0807436-1d6c-41c6-a66f-2c0c30c360fc.png)
we created airflow environment for test, it worked fine, but recently  when opening its UI, we got such weird message.
Appreciated if anyone could help!

**Comment by potiuk** [comment]
You need to contact MWAA support for that.
